.circleci/config.yml

```diff
@@ -39,21 +36,17 @@ jobs:
           command: |
             python manage.py check
       - run:
-          name: Black
+          name: Djhtml
           command: |
-            black . --check
-      - run:
-          name: Curly Lint
-          command: | 
-            curlylint .
+            git ls-files '*.html' | xargs djhtml --check
```

Comment:
Same comment as https://github.com/DemocracyClub/EveryElection/pull/2210#discussion_r1718178871 here for the same reasons